src/pandapipes/test/stanet_comparison/test_gas_stanet.py

```diff
@@ -48,8 +49,8 @@ def test_case_combined_3parallel_pc(use_numba, log_results=False):
     net = nw.gas_3parallel(method="pc")
     p_diff, v_diff_abs = pipeflow_stanet_comparison(net, log_results, friction_model="colebrook",
                                                     use_numba=use_numba)
-    assert np.all(p_diff < 0.002)
-    assert np.all(v_diff_abs < 0.03)
+    assert np.all(p_diff < 10 ** -4)
```

Comment:
Could we use the scientific representation here? I.e. 1e-4, I think that is more readable...